platforms/php/buildPhp.sh

```diff
@@ -114,7 +115,7 @@ buildPhpComposer() {
 		set -ex
 		composerDir="/opt/php-composer/$version"
 		mkdir -p "$composerDir"
-		export phpbin="/opt/php/$PHP81_VERSION/bin/php" 
+		export phpbin="/opt/php/$PHP_VERSION/bin/php" 
```

Comment:
why did we change this?

Reply:
This change is not necessary, anyways PHP81_VERSION is same as PHP_VERSION (in line 100 PHP81_VERSION is assigned to PHP_VERSION)

Reply:
let's revisit this, even if we want to use some version of php to build composer, let's use a newer version or see if there is another way to get composer